lib/models/product_model.dart

```diff
@@ -75,10 +72,7 @@ class Product {
         isPublished = map['isPublished'],
         category =
             map['category'] != null ? Category.values[map['category']] : null,
-        stockList = map['stockList']
-            ?.map<Stock>(
-                (stock) => Stock.fromMap(stock.cast<String, dynamic>()))
-            ?.toList(),
+        stock = map['stock'] != null ? Stock.fromMap(map['stock'].cast<String, dynamic>()) : null,
```

Comment:
It seems this line can be written in 2 lines.

Reply:
Can you please suggest how to write it in 2 lines? I'm not quite sure what you mean :)

Reply:
```
stock = map['stock'] != null ?
    Stock.fromMap( map['stock'].cast<String, dynamic>() ) : null
```

Reply:
Since this is less than the max line length, I don’t think this needs to be done. I won’t take this fix.